Userland/Libraries/LibGL/GLContext.h

```diff
@@ -406,6 +415,7 @@ class GLContext final {
             decltype(&GLContext::gl_color_material),
             decltype(&GLContext::gl_get_light),
             decltype(&GLContext::gl_clip_plane),
+            decltype(&GLContext::gl_get_clip_plane),
```

Comment:
You don't need to add this - we only do that if the invocation can be added to a list.